modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSEntries.tsx

```diff
@@ -439,6 +451,59 @@ const FDSEntries = ({
 		navigate(url);
 	};
 
+	const onDeleteClick = ({
+		itemData,
+		loadData,
+	}: {
+		itemData: FDSEntry;
+		loadData: Function;
+	}) => {
+		openModal({
+			bodyHTML: Liferay.Language.get(
+				'deleting-a-dataset-is-an-action-that-cannot-be-reversed'
+			),
+			buttons: [
+				{
+					autoFocus: true,
+					displayType: 'secondary',
+					label: Liferay.Language.get('cancel'),
+					type: 'cancel',
+				},
+				{
+					displayType: 'danger',
+					label: Liferay.Language.get('delete'),
+					onClick: ({processClose}: {processClose: Function}) => {
+						processClose();
+
+						fetch(itemData.actions.delete.href, {
+							method: itemData.actions.delete.method,
```

Comment:
Nice! This is probably better than using direct endpoint for deletion.

Reply:
Indeed!